File: python/Temp.py
class Temp:
    # deklarasi class yang berisi data mahasiswa

    # constructor
    def __init__(self, nim, name, prodi, fakultas):
        self.nim = nim
        self.name = name
        self.prodi = prodi
        self.fakultas = fakultas

    # getter NIM
    def getNim(self):
        return self.nim
    # setter NIM

    def setNim(self, nim):
        self.nim = nim
    # getter Name

    def getName(self):
        return self.name
    # setter Name

    def setName(self, name):
        self.name = name
    # getter Prodi

    def getprodi(self):
        return self.prodi
    # setter Prodi

    def setProdi(self, prodi):
        self.prodi = prodi
    # getter Fakultas

    def getfakultas(self):
        return self.fakultas
    # setter fakultas

    def setFakultas(self, fakultas):
        self.fakultas = fakultas

File: cpp/temp.cpp
#include <bits/stdc++.h>
#include <iostream>
#include <string>

using namespace std;

class Temp // deklarasi class yang berisi data mahasiswa
{
private:
    string nim, name, prodi, fakultas;

public:
    Temp() // constructor
    {
        this->nim = "";
        this->name = "";
        this->prodi = "";
        this->fakultas = "";
    }
    Temp(string nim, string name, string prodi, string fakultas) // constructor
    {
        this->nim = nim;
        this->name = name;
        this->prodi = prodi;
        this->fakultas = fakultas;
    }
    // getter Nim
    string getNim()
    {
        return nim;
    }
    // setter Nim
    void setNim(string nim)
    {
        this->nim = nim;
    }
    // getter nama
    string getName()
    {
        return name;
    }
    // setter nama
    void setName(string name)
    {
        this->name = name;
    }
    // getter prodi
    string getProdi()
    {
        return prodi;
    }
    // setter prodi
    void setProdi(string prodi)
    {
        this->prodi = prodi;
    }
    // getter fakultas
    string getFakultas()
    {
        return fakultas;
    }
    // setter fakultas
    void setFakultas(string fakultas)
    {
        this->fakultas = fakultas;
    }
    // destructor
    ~Temp()
    {
    }
};